lib/portlayer/exec/exec.go

```diff
@@ -40,6 +40,7 @@ var (
 )
 
 func Init(ctx context.Context, sess *session.Session, source extraconfig.DataSource, _ extraconfig.DataSink) error {
+	log.Info("Beginning initialization of portlayer exec component")
```

Comment:
I left this as info since it will only occur once on every start of the persona. It can be handy to see this log in customer log bundles since customers are likely not going to be running a debug version of VIC.